init.zsh

```diff
@@ -17,6 +17,51 @@ if ! autoload -Uz is-at-least || ! is-at-least "$min_zsh_version"; then
 fi
 unset min_zsh_version
 
+# zprezto convenience updater
+function zprezto-update {
+  function cannot-fast-forward {
+    local STATUS="$1"
+    [[ "${STATUS}" ]] && printf "%s\n" "${STATUS}"
+    printf "Unable to fast-forward the changes. You can fix this by "
+    printf "running\ncd '%s' and then\n'git pull' " "${ZPREZTODIR}"
+    printf "to manually pull and possibly merge in changes\n"
+  }
+  cd -- "${ZPREZTODIR}" || return 7
```

Comment:
This will still trigger chpwd callbacks (which we don't want to run). There's an option to cd which won't trigger them... Also, I'm a bit confused why pushd didn't work...